qutip/visualization.py

```diff
@@ -403,23 +455,22 @@ def sphereplot(theta, phi, values, fig=None, ax=None, save=False):
     r = array(abs(values))
     ph = angle(values)
     # normalize color range based on phase angles in list ph
-    nrm = mpl.colors.Normalize(ph.min(), ph.max())
+    norm = mpl.colors.Normalize(ph.min(), ph.max())
 
     # plot with facecolors set to cm.jet colormap normalized to nrm
     ax.plot_surface(r * xx, r * yy, r * zz, rstride=1, cstride=1,
-                    facecolors=cm.jet(nrm(ph)), linewidth=0)
-    # create new axes on plot for colorbar and shrink it a bit.
-    # pad shifts location of bar with repsect to the main plot
-    cax, kw = mpl.colorbar.make_axes(ax, shrink=.66, pad=.02)
+                    facecolors=cmap(norm(ph)), linewidth=0,)
 
-    # create new colorbar in axes cax with cm jet and normalized to nrm like
-    # our facecolors
-    cb1 = mpl.colorbar.ColorbarBase(cax, cmap=cm.jet, norm=nrm)
-    # add our colorbar label
-    cb1.set_label('Angle')
+    if colorbar:
```

Comment:
The `colorbar` is too close to the plot.

Reply:
This is for ```sphereplot```, right? I added ``` pad=.05```